middleware/kubernetes/autopath.go

```diff
@@ -55,7 +55,9 @@ func (apw *AutoPathWriter) overrideMsg(res *dns.Msg, force bool) error {
 		if apw.original.Name == a.Header().Name {
 			continue
 		}
-		res.Answer = append(res.Answer, newCNAME(apw.original.Name, dns.Fqdn(a.Header().Name), a.Header().Ttl))
+		res.Answer = append(res.Answer, nil)
```

Comment:
I actually googled up the same link.  But after reading comment ...

```
A better prepend, as it generates less garbage:

data = append(data, “”)
copy(data[1:], data)
data[0] = “Prepend Item”
```

I opted for the less garbagey method.  I did not verify if it produces less garbage, but it looks like it does.